### PR TITLE
Fix scrubbing freon\water vapor + fix labelling canisters with freon\water vapor

### LINKED ISSUE #20278: Scrubbers can't be set to scrub Freeon
You can't enable it, it's just an option you can't check.

### PR DESCRIPTION
**Pull request by zig-for**
:cl: 
fix: you can now scrub freon and water vapor
fix: you can now label canisters as freon and water vapor
/:cl:

Fixes #20278.

### CODE REVIEW

**Comment by A-t48** [comment]
@Iamgoofball

File: tgui/src/interfaces/atmos_filter.ract [hunk 1]
```diff
@@ -21,11 +21,11 @@
       action='filter' params='{"mode": "co2"}'>CO2</ui-button>
     <ui-button state='{{data.filter_type == "n2o" ? "selected" : null}}'
       action='filter' params='{"mode": "n2o"}'>N2O</ui-button>
-	<ui-button state='{{data.filter_type == "bz" ? "selected" : null}}'
+  	<ui-button state='{{data.filter_type == "bz" ? "selected" : null}}'
```
Comment (Shadowlight213) [inline tgui/src/interfaces/atmos_filter.ract]: What?

File: tgui/src/interfaces/airalarm/scrubbers.ract [hunk 2]
```diff
@@ -25,12 +25,12 @@
...
         <ui-button icon='{{filter_toxins ? "check-square-o" : "square-o"}}' style='{{filter_toxins ? "selected" : null}}'
           action='tox_scrub' params='{"id_tag": "{{id_tag}}", "val": {{+!filter_toxins}}}'>Plasma</ui-button>
-		<ui-button icon='{{filter_bz ? "check-square-o" : "square-o"}}' style='{{filter_bz ? "selected" : null}}'
-		  action='bz_scrub' params='{"id_tag": "{{id_tag}}", "val": {{+!filter_bz}}}'>BZ</ui-button>
-		<ui-button icon='{{filter_freon ? "check-square-o" : "square-o"}}' style='{{filter_freon ? "selected" : null}}'
-		  action='freon_scrub' params='{"id_tag": "{{id_tag}}", "val": {{+!filter_freon}}}'>Freon</ui-button>
-		<ui-button icon='{{filter_water_vapor ? "check-square-o" : "square-o"}}' style='{{filter_water_vapor ? "selected" : null}}'
-		  action='water_vapor_scrub' params='{"id_tag": "{{id_tag}}", "val": {{+!filter_water_vapor}}}'>Water Vapor</ui-button>
+		    <ui-button icon='{{filter_bz ? "check-square-o" : "square-o"}}' style='{{filter_bz ? "selected" : null}}'
+		      action='bz_scrub' params='{"id_tag": "{{id_tag}}", "val": {{+!filter_bz}}}'>BZ</ui-button>
```
Comment (Shadowlight213) [inline tgui/src/interfaces/airalarm/scrubbers.ract]: why are there just spaces?

File: tgui/src/interfaces/airalarm/scrubbers.ract [hunk 3]
```diff
@@ -25,12 +25,12 @@
           action='n2o_scrub' params='{"id_tag": "{{id_tag}}", "val": {{+!filter_n2o}}}'>N2O</ui-button>
         <ui-button icon='{{filter_toxins ? "check-square-o" : "square-o"}}' style='{{filter_toxins ? "selected" : null}}'
           action='tox_scrub' params='{"id_tag": "{{id_tag}}", "val": {{+!filter_toxins}}}'>Plasma</ui-button>
-		<ui-button icon='{{filter_bz ? "check-square-o" : "square-o"}}' style='{{filter_bz ? "selected" : null}}'
-		  action='bz_scrub' params='{"id_tag": "{{id_tag}}", "val": {{+!filter_bz}}}'>BZ</ui-button>
-		<ui-button icon='{{filter_freon ? "check-square-o" : "square-o"}}' style='{{filter_freon ? "selected" : null}}'
-		  action='freon_scrub' params='{"id_tag": "{{id_tag}}", "val": {{+!filter_freon}}}'>Freon</ui-button>
-		<ui-button icon='{{filter_water_vapor ? "check-square-o" : "square-o"}}' style='{{filter_water_vapor ? "selected" : null}}'
-		  action='water_vapor_scrub' params='{"id_tag": "{{id_tag}}", "val": {{+!filter_water_vapor}}}'>Water Vapor</ui-button>
+		    <ui-button icon='{{filter_bz ? "check-square-o" : "square-o"}}' style='{{filter_bz ? "selected" : null}}'
```
Comment (Shadowlight213) [inline tgui/src/interfaces/airalarm/scrubbers.ract]: So, what does this whitespace do?
This isn't even built

**Comment by Iamgoofball** [comment]
Fucking tgui thanks dude

**Pull request merged**